JSON column type

The JSON type was introduced in MySQL 5.7.8. Inserting values as string works fine, but selecting from the table returns this error:

```
--- FAIL: TestStorage (0.09s)
	storage_test.go:32: Unknown FieldType 245
```

I tried casting the JSON column to VARCHAR in the select but soon find out MySQL doesn't support it. Is there a workaround for dealing with JSON columns? Thanks!

--- a/const.go
+++ b/const.go
@@ -107,7 +107,8 @@ const (
 	fieldTypeBit
 )
 const (
-	fieldTypeNewDecimal byte = iota + 0xf6
+	fieldTypeJSON byte = iota + 0xf5
+	fieldTypeNewDecimal
 	fieldTypeEnum
 	fieldTypeSet
 	fieldTypeTinyBLOB

--- a/packets.go
+++ b/packets.go
@@ -1106,7 +1106,7 @@ func (rows *binaryRows) readRow(dest []driver.Value) error {
 		case fieldTypeDecimal, fieldTypeNewDecimal, fieldTypeVarChar,
 			fieldTypeBit, fieldTypeEnum, fieldTypeSet, fieldTypeTinyBLOB,
 			fieldTypeMediumBLOB, fieldTypeLongBLOB, fieldTypeBLOB,
-			fieldTypeVarString, fieldTypeString, fieldTypeGeometry:
+			fieldTypeVarString, fieldTypeString, fieldTypeGeometry, fieldTypeJSON:
 			var isNull bool
 			var n int
 			dest[i], isNull, n, err = readLengthEncodedString(data[pos:])

